src/devtools/client/debugger/src/actions/pause/previewPausedLocation.js

```diff
@@ -2,32 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
-//
-
 import { selectLocation } from "../sources";
-import { getContext, getSourceByURL } from "../../selectors";
-
-export function previewPausedLocation(location) {
-  return ({ dispatch, getState }) => {
-    const cx = getContext(getState());
-    const source = getSourceByURL(getState(), location.sourceUrl);
-    if (!source) {
-      return;
-    }
-
-    const sourceLocation = {
-      line: location.line,
-      column: location.column,
-      sourceId: source.id,
-    };
-    dispatch(selectLocation(cx, sourceLocation));
-
-    dispatch({
-      type: "PREVIEW_PAUSED_LOCATION",
```

Comment:
I think there are some additional places where `PREVIEW_PAUSED_LOCATION` is used

Reply:
i'll have to do some digging, it looks like there were two methods with the same dispatch type